ensembl/htdocs/info/genome/compara/epo_anchors_info.html

```diff
@@ -6,52 +6,44 @@
 
 <body>
 
-<h2>The EPO pipeline is split into three parts</h2>
+<p>
+In the EPO pipeline, Enredo, Pecan and Ortheus are the tip of the iceberg.
+They require homology and synteny information that are derived from alignments of <em>anchor</em> sequences.
+This page explains what the anchors are, how they are generated and used.
+</p>
 
-<h3>1). Anchor generation</h3>
+<h3 id="anchorgeneration">Anchor generation</h3>
 
 <p>
 The anchors represent short regions of conservation, typically about 100 bases in length, from a phylogenetically representative subset of the species we wish to align.
```

Comment:
In this section, since changes are being made: 

> phylogenetically representative

 does not have a clear meaning. Could it somehow be better defined as a related group of species with an (expected) level of similarity? What exactly is the criteria for selecting the anchoring species?

Reply:
We have not defined this. The idea was to eventually use some sort of clustering of the species based on their _mash_ distances so that all species in the alignment are close enough to a species in the anchor set (threshold to be defined). But this hasn't been done yet, and I don't think it will get done.
In practice, we've looked at the genome coverage of the EPO alignment